hardhat.config.js

```diff
@@ -0,0 +1,18 @@
+// Note: this config is only here as a matter of convenience to allow all hardhat commands that can be run from core
```

Comment:
this _might_ case issues. you now have effectively created nested hard hat packages. i.e the protocol is has a hardhat and the sub packages have hard hat. I _think_ that if you are within the child package (common/liquidator est.) then you'll use the child packages hard hat and if you are in root of the protocol then you'll use the `core` hard hat (as this one points to the `core` package. does that map to your understanding?